api/turing/api/ensemblers_api_it_test.go

```diff
@@ -1,3 +1,4 @@
+//go:build integration
```

Comment:
Why is this changed?

Reply:
Oh yeah tbh they probably got left there by my IDE or the linter, I should be able to remove them right away.

Reply:
Okay I've just checked, it was my IDE running gofmt automatically when I saved these files.